features/customers.feature

```diff
@@ -220,3 +220,114 @@ Scenario: Deactivate and Activate a Customer
     And I press the "Retrieve" button
     Then I should see the message "Success"
     And I should see "True" in the "Active" dropdown
+
+
+Scenario: Update a Customer
+    When I visit the "Home Page"
+    And I press the "Search" button
+    Then I should see the message "Success"
+    When I copy the "Id" field
+    And I set the "First Name" to "Test FN"
+    And I set the "Last Name" to "Test LN"
+    And I set the "Email" to "test@gmail.com"
+    And I set the "Password" to "test788pass"
+    And I select "True" in the "Active" dropdown
+    And I set the "Street" to "10th Casselberry Way"
+    And I set the "City" to "monroe"
+    And I set the "Pin Code" to " 07319"
+    And I set the "State" to "NJ"
+    And I set the "Country" to "USA"
+    And I press the "update" button
+    Then I should see the message "Success"
+    When I press the "clear" button
+    And I paste the "Id" field
+    And I press the "retrieve" button
+    Then I should see the message "Success"
+    And I should see "Test FN" in the "first_name" field
```

Comment:
Can you change the field names in this block as well according to UI?
~first_name~ First Name and so on.